CMakeLists.txt

```diff
@@ -87,18 +87,40 @@ option(OCIO_WARNING_AS_ERROR "Set build error level for CI testing" OFF)
 
 if(OCIO_BUILD_GPU_TESTS OR OCIO_BUILD_APPS)
     set(OCIO_GL_ENABLED ON)
-    find_package(OpenGL QUIET)
-    if(NOT OpenGL_FOUND)
-        message(WARNING "OpenGL not found.")
-        set(OCIO_GL_ENABLED OFF)
+
+    # Determine the vendor of GPU
+    find_package(CUDA QUIET)
+    if(CUDA_FOUND)
+        set(OCIO_IS_NVIDIA ON)
+    else()
+        set(OCIO_IS_NVIDIA OFF)
+        message(STATUS "CUDA not found")
+    endif()
+
+    # Link to libOpenGL.so if the GPU vendor is NVidia
+    # Nvidia's EGL implementation will be used
+    if(${OCIO_EGL_NVIDIA})
```

Comment:
Should that be a test on `OCIO_IS_NVIDIA`?

Reply:
Indeed. Thank you for pointing this out!